### ISSUE TITLE
Update default distribution to Xenial for Travis

### ISSUE THREAD

**Issue by bmorcos** [nengo/nengo-bones#72]
Currently the default distribution for Travis CI is set to Trusty (14.04) in the [template](https://github.com/nengo/nengo-bones/blob/master/nengo_bones/templates/.travis.yml.template#L11). This has reached EoL (or at least end of support) as of [April 2019](https://wiki.ubuntu.com/Releases).

You can specify the dist for jobs (i.e. `dist: xenial`), however this seems to interact poorly with the `apt-install` option. For example, in `nengo-pynq-hw` I had the following:

```
travis_yml:
  jobs:
    - script: static
    - dist: xenial
      apt_install:
        - clang-format
        - cppcheck
```

The build failed as the package `clang-format` could not be found and the [build log](https://travis-ci.com/nengo/nengo-pynq-hw/jobs/263482678) shows that apt packages were trying to be installed from the default Trusty 14.04 repos instead of the Xenial 16.04 repos, despite the `dist: xenial` option.

**TL:DR**
- Setting the Linux distro under `jobs:` does not affect the `apt_install` option. 
- The global default `dist: trusty` hit EoL and should be updated.
- Perhaps this global dist should be accessible from `.nengobones.yml` as well?

**Comment by tbekolay** [comment]
Done in #79.
